_posts/en/newsletters/2020-04-08-newsletter.md

```diff
@@ -156,8 +156,14 @@ FIXME:jnewbery (write or delete section)
 - [LND #4087][] Merge pull request #4087 from Crypt-iQ/wt_hs_0310 FIXME:moneyball
   FIXME: suggested link to https://github.com/lightningnetwork/lnd/blob/master/docs/watchtower.md#tor-hidden-services
 
-- [LND #4079][] Merge pull request #4079 from guggero/psbt-chanfunding FIXME:dongcarl
-  FIXME: link to https://github.com/lightningnetwork/lnd/blob/master/docs/psbt.md
+- [LND #4079][] adds support for funding channels with [Partially Signed
+  Bitcoin Transactions][topic psbt] (PSBTs), allowing any
+  PSBT-compatible wallet to fund a channel open. Previously, channel
+  funding was only possible with LND's internal wallet. Once a channel
+  has been funded, LND manages all other operations like normal.  Users
```

Comment:
perhaps s/funded,/funded with a PSBT,/

Reply:
I feel like there is a better way of writing "manages all other operations like normal" but am blanking atm on how :)